src/references/OBI/OBI_assaySubset.xml.rdf

```diff
@@ -8407,8 +8364,976 @@
         <rdfs:label>chromosome conformation capture sequencing assay</rdfs:label>
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0002458"/>
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0600047"/>
-        <IAO_0000111>chromosome conformation capture sequencing assay</IAO_0000111>
-        <IAO_0000412 rdf:resource="http://purl.obolibrary.org/obo/obi.owl"/>
+        <obo:IAO_0000111>chromosome conformation capture sequencing assay</obo:IAO_0000111>
+        <obo:IAO_0000412 rdf:resource="http://purl.obolibrary.org/obo/obi.owl"/>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/OBI_0002621 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_0002621">
```

Comment:
How do we know which assays to include in this list? Or is it an all encompassing list of assays and these have just been added to the ontology recently?